tests/acceptance/AdminSample.feature

```diff
@@ -96,9 +111,102 @@ Feature: admin page for samples
+    And I press the button "Confirm"
+    And I wait "1" seconds
+    Then I should not see "Please fix the following input errors:"
+
+  Scenario: display 1 warning message when create with wrong longitude below -180
+    Given I am on "/adminSample/create"
+    And I should see "Create"
+    When I fill in the field of "name" "Sample[species_id]" with "87676:Eucalyptus pauciflora"
+    And I fill in the field of "name" "Sample[attributesList]" with "longitude=\"-200.1234\""
+    And I press the button "Create"
+    And I wait "1" seconds
+    Then I should see "Attribute value for longitude doesn't match WGS84 decimal format"
+    And I should see "Are you sure you want to continue?"
+    And I press the button "Confirm"
+    And I wait "1" seconds
+    Then I should not see "Please fix the following input errors:"
+
+  Scenario: display 1 warning message when create with incorrect formatting
+    Given I am on "/adminSample/create"
+    And I should see "Create"
+    When I fill in the field of "name" "Sample[species_id]" with "87676:Eucalyptus pauciflora"
+    And I fill in the field of "name" "Sample[attributesList]" with "longitude=\"123.456.789\""
+    And I press the button "Create"
+    And I wait "1" seconds
+    Then I should see "Are you sure you want to continue?"
+    And I should see "Attribute value for longitude doesn't match WGS84 decimal format"
+    And I press the button "Confirm"
+    And I wait "1" seconds
+    Then I should not see "Please fix the following input errors:"
+
+  Scenario: display 1 extra message when create with extra symbol
+    Given I am on "/adminSample/create"
+    And I should see "Create"
+    When I fill in the field of "name" "Sample[species_id]" with "87676:Eucalyptus pauciflora"
+    And I fill in the field of "name" "Sample[attributesList]" with "longitude=\"123.4567°\""
+    And I press the button "Create"
+    And I wait "1" seconds
+    Then I should see "Are you sure you want to continue?"
+    And I should see "Attribute value for longitude doesn't match WGS84 decimal format"
+    And I press the button "Confirm"
+    And I wait "1" seconds
+    Then I should not see "Please fix the following input errors:"
+
+  Scenario: display 1 warning message when create with region for lat_lon
+    Given I am on "/adminSample/create"
+    And I should see "Create"
+    When I fill in the field of "name" "Sample[species_id]" with "87676:Eucalyptus pauciflora"
+    And I fill in the field of "name" "Sample[attributesList]" with "latitude=something"
+    And I press the button "Create"
+    And I wait "1" seconds
+    Then I should see "Are you sure you want to continue?"
+    And I should see "Attribute value for latitude doesn't match WGS84 decimal format. For geographic location (country, sea, region) use another attribute name"
+    And I press the button "Confirm"
+    And I wait "1" seconds
+    Then I should not see "Please fix the following input errors:"
+
```

Comment:
**suggestion:** If these tests are passing, should add `@ok` tag to them which would be useful when using the `acceptance_runner` to execute the tests.